### PR TITLE
(DOCSP-21088) Publishing mcli v1.23

### PR DESCRIPTION
**Pull request by JuliaMongo**

### CODE REVIEW

File: publishedbranches/docs-mongocli.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 prefix: 'mongocli'
 version:
   published:
-    - '1.23 (upcoming)'
-    - '1.22 (stable)'
+    - '1.24' (upcoming)''
```
Comment (melissamahoney-mongodb) [inline publishedbranches/docs-mongocli.yaml:4]:
```suggestion
    - '1.24 (upcoming)'
```

File: publishedbranches/docs-mongocli.yaml [hunk 2]
```diff
@@ -31,8 +32,9 @@ version:
     - '0.0.4'
     - '0.0.3'
   active:
-    - '1.23 (upcoming)'
-    - '1.22 (stable)'
+    - '1.24' (upcoming)''
```
Comment (melissamahoney-mongodb) [inline publishedbranches/docs-mongocli.yaml]:
```suggestion
    - '1.24 (upcoming)'
```

File: publishedbranches/docs-mongocli.yaml [hunk 3]
```diff
@@ -31,8 +32,9 @@ version:
     - '0.0.4'
     - '0.0.3'
   active:
-    - '1.23 (upcoming)'
-    - '1.22 (stable)'
+    - '1.24' (upcoming)'
```
Comment (melissamahoney-mongodb) [inline publishedbranches/docs-mongocli.yaml:35]:
```suggestion
    - '1.24 (upcoming)'
```

**Review by melissamahoney-mongodb — CHANGES_REQUESTED** [review]

File: publishedbranches/docs-mongocli.yaml (hunk 1 above)
Comment (JuliaMongo) [reply publishedbranches/docs-mongocli.yaml:4]: done

**Review by melissamahoney-mongodb — APPROVED** [review]

**Pull request merged**